CHANGELOG

```diff
@@ -1,9 +1,81 @@
-Version 2023.09 (unreleased)
+Version 2023.12 (unreleased)
   Friendica Core
+    Updates to the translations AR, BG, CA, CS, DE, EN GB, EN US, EO, ES, ET, FI, FR, GD, HU, IS, IT, JA, NL, PL, RO, RU, SV
+    Updates to the themes (frio, vier) [AlfredSK, annando, anubis2814, haheute, foss-, MrPetovan, Raroun, xundeenergie]
+    Updates to the documentation [AndyHee, BirdboyBolu, foss-, MrPetovan]
+    Fixed a bug in notification links [annando,  keithhacks]
+    Fixed a bug in content negotiation [GidiKroon]
+    Fixed the link in invitations [Raroun]
+    Fixed 0Auth connection with Buffer and redirection problems [annando, HankG]
+    Fixed a visibility bug with Calendar entries [MrPetovan]
+    Fixed a JsonLD parsing issue [annando]
+    Fixed confusion about nickname and username [MrPetovan, Raroun]
+    Fixed a problem with viewing remote contact profiles [MrPetovan]
+    Fixed a bug with previewing linked postings [annando]
+    Fixed a problem parsing Mastodon WebFingers [MrPetovan]
+    Fixed a bug that could lead to deleting your own account unintended [MrPetovan]
+    Fixed redirections in 2FA settings page [MrPetovan]
+    Fixed a problem with distributing some comments [annando]
+    Fixed a bug that caused public groups not being able to hide members [annando]
+    Fixed RSS/Atom feed of group accounts [MrPetovan]
+    Fixed a bug that prevented bi-directional delivery of DMs [annando]
+    Fixed a bug in privacy settings for remote-self [annando]
+    Fixed a bug that prevented the selection of photos with captions [Raroun]
+    Fixed a bug in Magic Auth [git-marijus]
+    Improved Bluesky connector core [annando]
```

Comment:
Shouldn't that be part in the addon section?

Reply:
You did add some Bluesky provisions in the core itself.